.github/workflows/e2e.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   scalar-ist-e2e-test:
     runs-on: ubuntu-latest
     env:
-      SCALARDL_VERSION: 3.6.0
+      SCALARDL_VERSION: 3.5.3
```

Comment:
What is the reason for downgrading?

Reply:
Sorry I forgot to mention the reason.

Because we are using `scalardl-java-client-sdk 3.5.3` for contracts_and_functions creation.
https://github.com/scalar-labs/scalar-ist/blob/feat/deploy-by-shell/contracts_and_functions/build.gradle#L19

I think we are planning to release Scalar IST project for each Scalar DL version, that is why I downgraded.